roles/master/tasks/main.yml

```diff
@@ -7,7 +7,7 @@
   when: kubeadm_kubernetes_version is defined
 
 - name: create flannel network
-  shell: kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/bc79dd1505b0c8681ece4de4c0d86c5cd2643275/Documentation/kube-flannel.yml
+  shell: kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/960b3243b9a7faccdfe7b3c09097105e68030ea7/Documentation/kube-flannel.yml
```

Comment:
changed to `960b3243b9a7faccdfe7b3c09097105e68030ea7`, the latest commit on master.